src/sentry/static/sentry/app/views/organizationIntegrations/integrationDetailedView.tsx

```diff
@@ -64,7 +64,15 @@ class IntegrationDetailedView extends AbstractIntegrationDetailedView<
   get alerts() {
     const provider = this.provider;
     const metadata = this.metadata;
-    const alerts = metadata.aspects.alerts || [];
+    const alerts = (metadata.aspects.alerts || []).map(item => {
+      switch (item.icon) {
+        case 'icon-warning':
+        case 'icon-warning-sm':
+          return {...item, icon: <IconWarning />};
+        default:
+          return {...item, icon: <IconFlag />};
+      }
+    });
```

Comment:
Should we add some comments about where these are coming from?

Reply:
Good idea.